efficientdet/model_inspect_v2.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+r"""Tool to inspect a model."""
```

Comment:
How about move this file to keras folder and no need to have v2.
After we fully verify this version, we will deprecate the original inference.py and model_inspect.py outside keras.

Reply:
Good.